hack/mk-manager-cluster.yaml

```diff
@@ -0,0 +1,4 @@
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+networking:
+  apiServerAddress: "172.17.0.1"
```

Comment:
where is this coming from? Why is it the same IP as the worker?

Reply:
If its updated at runtime, maybe we can put `"FILLED_AT_RUNTIME"` instead of a specific IP?